core/tests/test_utils.py

```diff
@@ -1445,9 +1445,10 @@ def save_new_topic_with_subtopic_schema_v1(
                 'name': name
             }])
 
-    def save_new_question(
+    def save_new_question( #pylint: disable=dangerous-default-value
             self, question_id, owner_id, question_state_data,
-            language_code=constants.DEFAULT_LANGUAGE_CODE):
+            language_code=constants.DEFAULT_LANGUAGE_CODE,
+            linked_skill_ids=[]):
```

Comment:
Don't use a default value here, pass it explicitly. You can move it before language_code.

Reply:
Done